src/main/java/Brad/Tasks/List.java

```diff
@@ -0,0 +1,133 @@
+package Brad.Tasks;
+
+import java.util.ArrayList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import Brad.Exceptions.dataCorruptedException;
+
+public class List {
+    private ArrayList<Task> inputList = new ArrayList<Task>();
+    private final String FILE_PATH = "data/data.md";
+
+    public void initializeFile()
```

Comment:
Can consider more OOP by adding a storage class that handles saving and loading from file